cron_ticket_email_parser.php

```diff
@@ -475,9 +477,11 @@ function getInboxFolder($client, $inboxNames) {
                     }
                 } elseif ($config_ticket_email_parse_unknown_senders)  {
                     // Parse even if the sender is unknown
-
-                    if (addTicket(0, 'Guest', $from_email, 0, $date, $subject, $message_body, $message->getAttachments(), $original_message_file)) {
-                        $email_processed = true;
+		    $bad_from_pattern = "/daemon|postmaster|reply|root|admin/i";
```

Comment:
I like this concept, but would probably omit 'reply' and 'admin'. 

We probably want automated emails from things like noreply@microsoft.com to be parsed

Some clients use admin@ for shared mailboxes.

Reply:
I agree.  I could see actually making the block list regex a configurable field later on in Settings.  Might take a stab at that later on.  I'll fix it now to just exclude daemon and postmaster.